src/components/Banner/Banner.tsx

```diff
@@ -170,11 +170,15 @@ export const Banner = ({
       <div className={clsx(styles['banner__textAndAction'])}>
         <div className={clsx(styles['banner__textContent'])}>
           {title && (
-            <Heading as={titleAs} size="h5">
+            <Heading as={titleAs} size="title-sm" variant="inherit">
```

Comment:
to match `main`, and necessary to make sure things like the NuxMegaphone display color correctly